smrf/distribute/image_data.py

```diff
@@ -62,6 +63,30 @@ def thread_variables(self):
 
         return self._thread_variables
 
+    @thread_variables.setter
+    def thread_variables(self, value):
+        self._thread_variables = value
+
+    def add_thread_variables(self, variables):
+        """Add a list or single variable to the thread variables
+
+        Args:
+            variables (list or str): List or string of variables to add
+        """
+
+        if isinstance(variables, str):
+            variables = list((variables, ))
```

Comment:
When I first saw this, I was curious of there is an alternative to this approach.
SO suggested: `variable.split()`

Just for reference

Reply:
So if it was a list already, then it would just output another list?

Reply:
No, it's to cast a string to a list.
```python
'foo'.split()
['foo']

```

Reply:
Ah, that would make it a bit cleaner.